metakb/transform/base.py

```diff
@@ -128,6 +254,17 @@ def _get_document_id(**parameters) -> str:
         blob = json.dumps(params_sorted).encode("ascii")
         return f"document:{sha512t24u(blob=blob)}"
 
+    @staticmethod
+    def _get_digest_for_str_lists(l: List[str]) -> str:  # noqa: E741
+        """Create digest for a list of strings
+
+        :param List[str] l: List of strings to get digest for
+        :return: Digest
+        """
+        l.sort()
```

Comment:
low impact but I think avoiding "l" as a variable name is probably a good practice (maybe you could use "strings" as a name instead)

Reply:
Ugh I get these warnings and ignore them. Naming is hard.

Reply:
Renaming to `str_list`